blockchain/blockchain-wallet-v4-frontend
# feat(settings): moved settings profile and changed to be trading limits and as Flyout

## Description (optional)
This PR implements flyout for trading limits page
![image](https://user-images.githubusercontent.com/67264242/106210840-e5fecf80-61c7-11eb-8849-851c4a7951e6.png)


## Testing Steps (optional)
Login and click on gear icon on top right corner, then click on Trading Limits from dropdown, new flyout should be visible
